conf/app.routes

```diff
@@ -1,4 +1,8 @@
 # microservice specific routes
 
-GET        /login             controllers.UserAccount.loginPage
-POST       /login             controllers.UserAccount.authenticateAction
+GET        /login             controllers.AccountController.loginPage
+POST       /login             controllers.AccountController.authenticateAction
+
```

Comment:
again do we need Action in the name of the controller methods.
they can be 
controllers.AccountController.authenticate
controllers.AccountController.logout